// Copyright 2018 The Chromium OS Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// Common definitions for reuse in other proto files.

syntax = "proto3";

package diagnostics.grpc_api;

// Empty message. Used in cases when an RPC doesn't have input parameters or
// doesn't return anything.
message EmptyMessage {}
